src/erlcloud_elb.erl

```diff
@@ -116,28 +139,380 @@ configure_health_check(LB, Target, Config) when is_list(LB) ->
                        [{"LoadBalancerName", [LB]},
                         {"HealthCheck.Target", Target}]).
 
-
-describe_load_balancer(Name) ->
+%% --------------------------------------------------------------------
+%% @doc describe_load_balancer with a specific balancer name or with a 
+%% specific configuration and specific balancer name. 
+%% @end
+%% --------------------------------------------------------------------
+describe_load_balancer(Name) when is_list(Name) ->
     describe_load_balancer(Name, default_config()).
 describe_load_balancer(Name, Config) ->
     describe_load_balancers([Name], Config).
 
-
-describe_load_balancers(Names) ->
-    describe_load_balancers(Names, default_config()).
+%% --------------------------------------------------------------------
+%% @doc Calls describe_load_balancer([], default_configuration())
+%% @end
+%% --------------------------------------------------------------------
+describe_load_balancers() ->
+    describe_load_balancers([], default_config()).
+
+%% --------------------------------------------------------------------
+%% @doc describe_load_balancers with specific balancer names or with a 
+%% specific configuration.
+%% @end
+%% --------------------------------------------------------------------
+describe_load_balancers(Names) when is_list(Names) ->
+    describe_load_balancers(Names, default_config());
+describe_load_balancers(Config = #aws_config{}) ->
+    describe_load_balancers([], Config).
+
+%% --------------------------------------------------------------------
+%% @doc Get descriptions of the given load balancers.
+%%      The account calling this function needs permission for the
+%%      elasticloadbalancing:DescribeLoadBalancers action.
+%% 
+%% Returns {{paged, NextPageId}, Results} if there are more than
+%% the current maximum count of results, {ok, Results} if everything
+%% fits and {error, Reason} if there was a problem.
+%% @end
+%% --------------------------------------------------------------------
+-spec describe_load_balancers(list(string()), aws_config()) -> 
+                             {ok, term()} | {{paged, string()}, term()} | {error, term()}.
 describe_load_balancers(Names, Config) ->
```

Comment:
Looks like this isn't backwards compatible with the previous implementation of `describe_load_balancers`. I don't want to break existing clients. I'd be ok with changing the name (`describe_load_balancers2`) or taking all these changes as a new module (`erlcloud_elb2`).